modelica_builder/model.py

```diff
@@ -194,6 +197,38 @@ def add_parameter(self, type_, identifier, arguments=None, assigned_value=None,
 
         self.add(parameter.transformation())
 
+    def update_model_annotation(self, modifications):
+        """Updates the model annotation modifications. If a modification exists
+        then it is updated, if it doesn't then it is inserted. If this method is
+        called more than once, previous calls are IGNORED and only the last call
+        is used.
+
+        The modifications param is a dictionary. Each key represents a modification
+        argument name. Each value represents the modification value. If a value
+        in the dict is another dict, then the modification is interpreted as a
+        class modification. If the key 'OVERWRITE_MODIFICATIONS' is found in a dict,
```

Comment:
Looks like this is "if the key 'OVERWRITE_MODIFICATIONS' is found in a dict with a True value, ...", correct?

Reply:
ah yeah, that's correct. It must be truthy. Let me update the docs